oc_cdtapi/OnecAPI.py

```
@@ -0,0 +1,229 @@
+        
+        Returns:
+            Tuple of (parsed_clients, error_message)
+        """
+        try:
+            clients_data = response.json()
+            
+            sorted_clients = sorted(clients_data, key=itemgetter('code'))
+            logging.debug("Successfully retrieved %d clients", len(sorted_clients))
+            return sorted_clients, None
+            
+        except KeyError as e:
```

Ruff (0.15.4)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)